packages/amplify-e2e-core/src/init/amplifyPush.ts

```diff
@@ -112,3 +145,19 @@ export function amplifyPushLayer(
     });
   });
 }
+
+export function amplifyPushIterativeRollback(cwd: string, testingWithLatestCodebase: boolean = false) {
+  return new Promise((resolve, reject) => {
+    spawn(getCLIPath(testingWithLatestCodebase), ['push', '--iterative-rollback'], { cwd, stripColors: true })
+      .wait('Are you sure you want to continue?')
+      .sendLine('y')
+      .wait(/.*/)
```

Comment:
I dont think we need this `wait` as the callback will be never called till child process completes